app/assets/javascripts/controllers/ops/diagnostics_database_form_controller.js

```diff
@@ -141,5 +161,19 @@ ManageIQ.angular.app.controller('diagnosticsDatabaseFormController', ['$http', '
     miqService.sparkleOff();
   }
 
+  function diagnosticsLogProtocol(prefix) {
+    var protocol = "";
+    if (prefix === 'nfs') {
```

Comment:
This could be a swtich.
```js
protocol = {
smb: 'Samba',
nfs: 'Network File System',
s3: 'AWS S3',
swift: 'OpenStack Swift'
}[prefix] || ''
```